packages/e2e-tests/helpers/index.ts

```diff
@@ -53,8 +53,12 @@ export async function isDevToolsTabActive(page: Page) {
   return classes.includes("active");
 }
 
-export async function startTest(page: Page, example: string, apiKey?: string) {
-  const recordingId = exampleRecordings[example];
+export async function openRecording(
+  page: Page,
+  example: string,
```

Comment:
nit: I know you didn't add this, but as I'm reading this code again I realize that "example" isn't a great name here.

We could probably also make this TypeScript-typed to avoid accidentally passing the wrong thing too?

```diff
diff --git a/packages/e2e-tests/helpers/index.ts b/packages/e2e-tests/helpers/index.ts
index 7ac98befd..74041005f 100644
--- a/packages/e2e-tests/helpers/index.ts
+++ b/packages/e2e-tests/helpers/index.ts
@@ -4,12 +4,11 @@ import dotenv from "dotenv";
 
 import { RecordingTarget } from "replay-next/src/suspense/BuildIdCache";
 
+import exampleRecordings from "../examples.json";
 import { debugPrint, getElementClasses } from "./utils";
 
 dotenv.config({ path: "../../.env" });
 
-const exampleRecordings = require("../examples.json");
-
 export async function getRecordingTarget(page: Page): Promise<RecordingTarget> {
   return page.evaluate(async () => {
     return (window as any).app.store.getState().app.recordingTarget;
@@ -53,8 +52,10 @@ export async function isDevToolsTabActive(page: Page) {
   return classes.includes("active");
 }
 
-export async function startTest(page: Page, example: string, apiKey?: string) {
-  const recordingId = exampleRecordings[example];
+type Key = keyof typeof exampleRecordings;
+
+export async function startTest(page: Page, exampleKey: Key, apiKey?: string) {
+  const recordingId = exampleRecordings[exampleKey];
   const base = process.env.PLAYWRIGHT_TEST_BASE_URL || "http://localhost:8080";
 
   let url = `${base}/recording/${recordingId}?e2e=1`;
diff --git a/packages/e2e-tests/tsconfig.json b/packages/e2e-tests/tsconfig.json
index 95323d5c6..f1f470c36 100644
--- a/packages/e2e-tests/tsconfig.json
+++ b/packages/e2e-tests/tsconfig.json
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "downlevelIteration": true,
     "esModuleInterop": true,
+    "resolveJsonModule": true,
     "types": ["cypress"]
   }
 }
```

Reply:
Thanks for the diff.